cli/tests/pcluster/models/test_imagebuilder.py

```diff
@@ -439,7 +439,7 @@ def test_get_stack_events(self, image_builder, mocker, stack_exists, expected_er
                     "ResourceType": "AWS::CloudFormation::Stack",
                     "StackId": "arn:aws:cloudformation:us-east-1:000000000000:stack/img",
                     "StackName": "img",
-                    "Timestamp": datetime.datetime(2021, 7, 13, 2, 20, 20, 000000, tzinfo=tz.tzutc()),
+                    "Timestamp": datetime.datetime(2021, 7, 13, 2, 20, 20, 0o00000, tzinfo=tz.tzutc()),
```

Comment:
Is this something expected?

Reply:
https://github.com/aws/aws-parallelcluster/security/code-scanning/581